package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "vue-pdf-embed",
-  "version": "1.1.6",
+  "version": "1.1.7",
```

Please revert this line for now (I will update it when creating a Git tag)

```suggestion
  "version": "1.1.6",
```